ci/jenkins/test.sh

```diff
@@ -804,7 +810,10 @@ function run_e2e {
 
     mkdir -p "${WORKDIR}/.kube"
     mkdir -p "${WORKDIR}/.ssh"
-    cp -f "${WORKDIR}/kube.conf" "${WORKDIR}/.kube/config"
+
+    if [[ $TESTBED_TYPE != "kind" ]]; then
+        cp -f "${WORKDIR}/kube.conf"  ${KUBECONFIG_PATH}
```

Comment:
For jobs which don't define `KUBECONFIG_PATH`, the path will be changed from "${WORKDIR}/.kube/config" to "${WORKDIR}/kube.conf".

Reply:
if you are passing  `KUBECONFIG_PATH` to this script then that case destination path will be different from  `${WORKDIR}/.kube/config` but if you not passing `KUBECONFIG_PATH` then destination path will be `${WORKDIR}/.kube/config`. 
could you correct me?

Reply:
@KMAnju-2021 @XinShuYang 
There are two cases here, 
1.KUBECONFIG_PATH is provided by user to the script then cp -f "${WORKDIR}/kube.conf"  ${KUBECONFIG_PATH} holds good,
2.if not then ${WORKDIR}/.kube/config should remain intact wrt assigned KUBECONFIG_PATH = ${WORKDIR}/.kube/config

Ideally
cp -f "${WORKDIR}/kube.conf"  ${KUBECONFIG_PATH} is only applicable if kube config is provided by user